services/movie.py

```diff
@@ -6,9 +7,13 @@
 def get_movies(
     genres_ids: list[int] = None,
```

Comment:
Using `list` as a default argument value is not recommended because it is mutable. Consider using `None` as the default value and then initializing it inside the function if needed. For example, use `genres_ids: list[int] = None` and then `genres_ids = genres_ids or []` inside the function.